SQL/0000-00-01-Permission.sql

```diff
@@ -131,3 +135,28 @@ CREATE TABLE `notification_modules_perm_rel` (
 -- populate notification perm table
 INSERT INTO notification_modules_perm_rel SELECT nm.id, p.permID FROM notification_modules nm JOIN permissions p WHERE nm.module_name='media' AND (p.code='media_write' OR p.code='media_read');
 INSERT INTO notification_modules_perm_rel SELECT nm.id, p.permID FROM notification_modules nm JOIN permissions p WHERE nm.module_name='document_repository' AND (p.code='document_repository_view' OR p.code='document_repository_delete');
+
+
+CREATE TABLE `role` (
+ `RoleID` INTEGER unsigned NOT NULL AUTO_INCREMENT,
+ `Name` varchar(255),
+ `Label` varchar(255),
```

Comment:
Could these 2 attributes be less than 255? It is very long for a label and ever more of a name.

Reply:
Yes... suggestions ?

Reply:
What do you think the maximum Name would be? Is 15-20 realistic?
For Label, maybe 5 or 10 char extra as it as to be correct english.

Reply:
the issue is.... some of our current production databases are on previous versions of mysql which do not have STRICT mode enabled. That means that no error is thrown if data is inserted with a higher char count than the varchar which means that data is truncated and lost.

Untill all prods are on 5.7+ I don't think we should risk shortening the char count at the deetriment of data

Reply:
10-15 also isn't realistic, with the existing permission codes we already have at least 32 character long ones:

`mysql> select max(length(code)) from permissions;
+-------------------+
| max(length(code)) |
+-------------------+
|                32 |
+-------------------+
1 row in set (0.01 sec)
`

Reply:
stuck on historical data :-( 
Those length should also be enforced in JS, like stopping the editing of the field when the limit is reach